config/config.go

```diff
@@ -478,6 +484,7 @@ type Config struct {
 	Identity                 security.Identity                   `yaml:",omitempty"`
 	InCluster                bool                                `yaml:"in_cluster,omitempty"`
 	InstallationTag          string                              `yaml:"installation_tag,omitempty"`
+	IstioConfigHelpMessages  map[string][]IstioConfigHelp        `yaml:"istio_config_help_messages,omitempty" json:"help,omitempty"`
```

Comment:
I know this is just a draft, but I was looking at this and wondering -- do we really want this in the config/ConfigMap? Is this something that is user-configurable? Or shouldn't this be somewhere outside of the configmap - this does not seem like something that is user-configurable so doesn't seem like it belongs in here.

Reply:
Well, this wasn't the first place for this information, before it was at the models package. 

I guess based on suggestions, it might be configurable by users, as it is right now, is not, I guess we need to modify the operator (does the operator overwrite these fields or leave it if is not something it manages?).

Anyway, yes, this is a draft and I was testing this approach, for me it seems the same, that does not belong here, but I'm open to suggestions and feedback. 

Thanks.

Reply:
yeah, this would require changes to the operator and the example crd/cr. I really don't see a need for this to be user-configurable. extra work for something that will be rarely (if ever) used.

Reply:
I agree, I would not make this configurable unless there was an actual RFE asking for that ability.  It's not clear to me that it's necessary in the initial feature.

Reply:
If we don't want to put in the config because of the implications with the operator, that's fine.
But it's unclear to me that this information that would easily grow and change should be hard-coded.
An option can be to use a YAML that can be easily updated.

Most of this info comes from the istio.io site and in the future, the idea is that it could be automatically generated in some way (i.e. a script that generates a map).

Reply:
I`m -1 for including these messages in config too, these messages should be not a user-facing config, and it increases the complexity of current configuration. 
We could use a static json/yaml to record these info and like @lucasponce said, we can use some scripts to keep this file updated in the future.
And I suggest embedding this file directly into binary, which is a new feature in golang 1.16, make it easy to manage these static files:)

Reply:
+1
Not a blocker, perhaps these definitions will not change and these can be just constants.